### ISSUE TITLE
On Windows 8.1, after waking up from sleep mode synergy service tries to start 2nd synergy server.

### ISSUE THREAD

**Issue by Fsmv** [synergy/synergy#4503]
I'm not totally sure how to reproduce this, but sometimes (maybe 10%?) when I put my Windows 8.1 machine running the synergy server sometimes when I resume the synergy application tries to start up a new service or something but can't because the service is already running. I then can't accept any connections until I go into the task manager and tell windows to restart the synergy service. Below is the log for when this error started. The part between the second failure and the third is then repeated 800 times until I noticed and restarted the service.

```
INFO: suspend
WARNING: detected application not running, pid=6312
INFO: backing off, wait=2s, failures=1
INFO: starting new process
activeDesktop:Winlogon
ERROR: could not get session id for process id 8216
INFO: starting new process
INFO: drag and drop enabled
ERROR: failed to init synwinhk.dll, another program may be using it
INFO: restarting your computer may solve this error
FATAL: failed to start server: unable to open screen
ERROR: failed to launch, error: process immediately stopped
INFO: backing off, wait=4s, failures=2
INFO: starting new process
activeDesktop:Winlogon
ERROR: could not get session id for process id 2228
INFO: starting new process
INFO: drag and drop enabled
ERROR: failed to init synwinhk.dll, another program may be using it
INFO: restarting your computer may solve this error
FATAL: failed to start server: unable to open screen
ERROR: failed to launch, error: process immediately stopped
INFO: backing off, wait=6s, failures=3
```

**Comment by samadhi82** [comment]
Same issue here. Both stable x64 and x86 versions (1.7.2) on Windows 8.1 are dumping with this error. I am, however, resuming from a sleep or hibernation state. My PC is constantly on. This was not an issue for me on any previous version.  

INFO: starting new process
synergys.exe: address not found for: Internet
Try `synergys.exe --help' for more information.
WARNING: detected application not running, pid=6364
INFO: backing off, wait=2s, failures=1
INFO: starting new process
activeDesktop:Default
INFO: starting new process
synergys.exe: address not found for: Internet
Try `synergys.exe --help' for more information.
WARNING: detected application not running, pid=7044
INFO: backing off, wait=2s, failures=1

**Comment by Evolter** [comment]
Hi, it happened to me as well, but only once. It happened on Windows 7, after Sleep mode. Version of Synergy (server): 1.6.2

**Comment by XinyuHou** [comment]
Solved by #4651
